src/views/assets/Cloud/Strategy/components/ActionInput.vue

```diff
@@ -157,6 +156,7 @@ export default {
             el: {
               hidden: true,
               settingReadonly: true,
+              showSetting: () => false,
               choices: []
             }
           }
```

Comment:
It seems there was an error in the text you provided, as it looks like it got cut off or is not coherent with respect to previous comments. Could you please rephrase this query?